src/services/metrics-recorder.ts

```diff
@@ -213,7 +213,7 @@ export class MetricsRecorder {
       const timeStreamMeasure = {
         Dimensions: timeStreamDimensions,
         MeasureName: 'elapsedTime',
-        MeasureValue: `${parseFloat(elapsedTime.toFixed(4))}`,
+        MeasureValue: `${parseFloat(elapsedTime.toString())}`,
```

Comment:
wrong time :D  granularity of the timestamp, not this one

Reply:
![image](https://user-images.githubusercontent.com/16605170/141499764-288173ee-4691-4e7f-a6e9-6fd06e5d4f87.png)

Reply:
That time on the right col

Reply:
on that case both can be changed, according to the error:

```
  {
    Reason: 'A record already exists with the same time, dimensions, measure name, and record version. A higher record version must be specified in order to update the measure value. Specifying record version is supported by the latest SDKs.',
    RecordIndex: 0
  }
```
dimension values are also taken into account, but that's good so even more granularity can be achieved, will do the change

Reply:
After a bit of research came to the conclusion that nodejs doesn't natively supports microseconds and after trying a few recomendations they didn't work reliably, recommend at the moment merge with only this change to provide slightly better granularity